lib/mongoid/fields/field_types.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+module Mongoid
+  module Fields
+
+    # Singleton module which contains a mapping of field types to field class.
+    # Custom field types can be configured.
+    #
+    # @api private
+    module FieldTypes
+
+      # The default mapping of field type symbol/string aliases to classes.
+      #
+      # @api private
+      DEFAULT_ALIASES = {
```

Comment:
This is a mapping, not an alias. Alias has a specific meaning in Ruby. It was called a mapping previously and I don't think we should change it.

Reply:
It's a mapping from aliases to classes, as per the comment.

Reply:
Yes, the constant itself is a *mapping* so its name should be `DEFAULT_MAPPING` (or `DEFAULT_SYMBOL_TO_CLASS_MAPPING`, but that's rather verbose and not necessary given the context.)